localclient/terminal_player.go

```diff
@@ -301,15 +316,8 @@ func (tc *terminalClient) makeRequest(method, apiURL string, data io.Reader, hea
 
 func (tc *terminalClient) createPlayer() error {
 	username, name := tc.getName()
-	var reqData = network.CreatePlayerModel{Username: username, DisplayName: name}
-	b, err := json.Marshal(reqData)
-	if err != nil {
-		return err
-	}
-	header := http.Header{
-		`Content-Type`: []string{`application/json`},
-	}
-	respBytes, err := tc.makeRequest(`POST`, `/create/player`, bytes.NewReader(b), header)
+	reqData := model.Player{ID: model.PlayerID(username), Name: name}
```

Comment:
🌵 📖 I'm surprised there isn't a linter to ensure that each field is on a newline. We should be doing that for our structs

Reply:
Or a go fmt setting or something... _shrug_